tools/server-spring/src/test/java/org/eclipse/rdf4j/http/server/repository/transaction/TransactionStartControllerTest.java

```diff
@@ -10,15 +10,24 @@
  *******************************************************************************/
 package org.eclipse.rdf4j.http.server.repository.transaction;
 
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.fail;
+import static org.assertj.core.api.Assertions.*;
+import static org.eclipse.rdf4j.common.transaction.IsolationLevels.SNAPSHOT;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
```

Comment:
Same here with the static imports.

Reply:
made this change but note isolationlevel.getvalue() is implemented through toString().